### PR TITLE
Add debug lig and update tests for user creation issue

### PR DESCRIPTION
**Pull request by RezaBG**

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -39,7 +40,8 @@ def run_migrations_offline() -> None:
     script output.
 
     """
-    url = config.get_main_option("sqlalchemy.url")
+    url = os.getenv("DATABASE_URL", config.get_main_option("sqlalchemy.url"))
```
Comment (arash-d) [inline alembic/env.py:43]: Why not using a constant for `"sqlalchemy.url"` and use it in a env file or constant.py file? the same for other places which are used

File: tests/test_tasks.py [hunk 2]
```diff
@@ -1,35 +1,49 @@
...
-    #
-    await client.post(
-        "/users", json={"username": "testuser", "email": "testuser@example.com", "password": "password123"}
+@pytest_asyncio.fixture
+async def setup_task(client):
+    # Create the task
+    response = await client.post(
+        "/tasks", json={"title": "Test Task", "description": "Task Description", "owner_id": 1}
     )
+    print(f"Task creation response: {response.status_code}")
```
Comment (arash-d) [inline tests/test_tasks.py:10]: use logger rather print

File: tests/test_tasks.py [hunk 3]
```diff
@@ -1,35 +1,49 @@
...
     assert response.json()["title"] == "Test Task"
 
 @pytest.mark.asyncio
-async def test_update_task(client):
+async def test_update_task(client, setup_task):
+    task_id = setup_task["id"]
     response = await client.put(
-        "/tasks/1",
-        json={"title": "Update Test Task", "description": "Updated task description", "owner_id": 1},
+        f"/tasks/{task_id}",
```
Comment (arash-d) [inline tests/test_tasks.py:39]: what happens if someone request a url with wrong id? did you consider it in your logic? if yes why you do not have test for that? same for update, delete, get methods

File: alembic/env.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
 from logging.config import fileConfig
```
Comment (arash-d) [inline alembic/env.py:1]: in all youor code not just here in test, try to have consistent formatting. read and search about black, sort, ruff, pylint and format your codes with these tools

Comment (RezaBG) [reply alembic/env.py]: I tried to apply it, could you check again?

File: alembic/env.py (hunk 1 above)
Comment (RezaBG) [reply alembic/env.py:33]: Could you check again?

**Review by IshanRastogi — APPROVED** [review]

**Pull request merged**